mutalyzer/GenRecord.py

```diff
@@ -312,6 +312,21 @@ def findGene(self, name) :
         return None
     #findGene
 
+    def getInfoByTranscriptID(self, transcript_id):
+        """
+        Returns a tuple with gene name and transcript name (i.e. its
+        `v-number') for a given transcript ID.
+
+        @param transcript_id:
+        @return: tuple
+        """
```

Comment:
I know it would be a pretty unique function in Mutalyzer, but could you complete the docstring? (Include param type and description, use e.g., `tuple(str, int)`.)